tiptapy/__init__.py

```diff
@@ -104,6 +107,29 @@ class Doc(BaseContainer):
     type = "doc"
 
 
+class OrderedList(ListItem, Paragraph):
+    type = "ordered_list"
+    wrap_tag = "ol"
+
+    def inner_render(self, node):
```

Comment:
@Pradhvan I think we don't need to override `inner_render`. cc: @shon

Reply:
Just tested without overriding the `inner_render` function. Manually tested with some sample JSON and even the tests pass. Looks like I was reinventing the wheel.  Pushing the updated changes.